pkg/operator/render_test.go

```diff
@@ -68,7 +68,7 @@ func TestIPFamilies(t *testing.T) {
 		Output: mcfgv1.IPFamiliesDualStack,
 	}, {
 		Ranges: []string{"2001:db8::/32", "192.168.2.0/20"},
-		Output: mcfgv1.IPFamiliesDualStack,
+		Output: mcfgv1.IPFamiliesDualStackIPv6Primary,
```

Comment:
Not introduced by your change, but as you are touching this file, can we maybe add the following 2 scenarios

* `[]string{ipv4, ipv4}`
* `[]string{ipv6, ipv6}`

with `Error: true`? Just to make sure if someone in the future adds a handler for this, they are forced to fix the test

Reply:
Those are both valid cases. You're allowed to have multiple CIDRs per ip version. We could add more test cases to ensure they're handled right though.

Reply:
ACK, thanks for explaining! I always thought that we only allow single subnet per IP stack